### PR TITLE
gtfs->netex conversion unit test and circleci mods

### PR DESCRIPTION
**Pull request by philosotractor**
# Added
 *   test: netex: unit test for gtfs->netex conversion
    netex: add testdata, gtfs input zip and netex reference result zip
    util: file namespace created for file and stream helpers, slurp-bytes

# Environment
  *  chouette shell tool added to circleci docker image
     circleci config.edn overwriting before test execution

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,8 +50,16 @@ jobs:
       - run: aws s3 cp s3://ote-assets/places/spatial_relations_places.csv database/spatial_relations_places.csv
       - run: cd database && sh circleci_setup.sh
 
+      # setup ote config
+
```
Comment (philosotractor) [inline .circleci/config.yml]:
```suggestion

```

Comment (philosotractor) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -46,8 +50,16 @@ jobs:
       - run: aws s3 cp s3://ote-assets/places/spatial_relations_places.csv database/spatial_relations_places.csv
       - run: cd database && sh circleci_setup.sh
 
+      # setup ote config
```
Comment (philosotractor) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -46,8 +50,15 @@ jobs:
       - run: aws s3 cp s3://ote-assets/places/spatial_relations_places.csv database/spatial_relations_places.csv
       - run: cd database && sh circleci_setup.sh
 
+
       # run tests!
-      - run: cd ote && lein test
+      - run:
+          name: Setup config.edn for tests
```
Comment (rckopteri) [inline .circleci/config.yml]: 👍

File: ote/src/cljc/ote/util/file.cljc [hunk 4]
```diff
@@ -0,0 +1,12 @@
+(ns ote.util.file
```
Comment (rckopteri) [inline ote/src/cljc/ote/util/file.cljc]: Would there be a case when ui is calling these util functions? 
I think that this file could be in clj folder

Comment (philosotractor) [reply ote/src/cljc/ote/util/file.cljc]: Added commit 1323685

**Pull request merged**